RELEASING.md

```diff
@@ -4,35 +4,35 @@ The release process for SNAPSHOT versions is handled by the `after_success.sh` s
 
 ### To push a release version
 
-1. Make sure the team agrees that it is time to release. Verify that all unit and integration tests for the last commit have passed.
+1. Make sure the team agrees that it is time to release. 
 
-2. Run `utilities/update_versions.sh` from the repository's base directory.
-This script takes optional arguments denoting the new versions for each qualifier (alpha, beta and/or GA). By default, if the current version is X.Y.Z-SNAPSHOT, the script will update the version in all the pom.xml and other relevant files to X.Y.Z. Please refer to the documentation in `utilities/update_versions.sh` for more details. Commit this version locally:
+2. Look over all of the commits since the last release and make sure there are no breaking changes on the public surface. If there are any breaking changes, create and merge a new PR to revert the surface back.
```

Comment:
Technically this asks to re-review all commits done since last release. This is simply impossible. Should it be instead a recommendation for each contributor before making a PR, described either in GA libraries readmes and/or in the CONTRIBUTING.md doc?

Another approach would be to add some automation on top of that (like running old tests against newer jar - we have most of that infrastructure as part of GAE work done, and it doesn't have to be automated, can be a manual run)